File: mario/mario.c
#include <cs50.h>
#include <stdio.h>
void pyramid(int x);
int main(void)
{
    int x;
    do {
        x = get_int("Height:");
    }
    while(x < 1 || x > 8);
    pyramid(x);

}

void pyramid(int x){
    for(int i =0;i<x;i++){
        for(int j =x-i-1;j>0;j--){
            printf(" ");
        }
        for(int j=0;j<=i;j++){
            printf("#");
        }
        printf("  ");
        for(int j=0;j<=i;j++){
            printf("#");
        }
        printf("\n");
    }
}

File: mario(2)/mario.py
from cs50 import get_int

i = get_int("Height: ")
while not (i>= 1 and i <=8):
    i = get_int("Heigth: ")


for a in range(i):
    for b in range(i-a-1):
        print(" ",end = "")
    for c in range(a+1):
        print("#",end = "")
    print("")